### PR TITLE
xorg.xf86inputlibinput: 1.0.0 -> 1.0.1

### PR DESCRIPTION
**Pull request by TredwellGit**
###### Motivation for this change
https://lists.x.org/archives/xorg-announce/2021-April/003083.html

###### Things done
- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/x11/xorg/default.nix [hunk 1]
```diff
@@ -1717,11 +1717,11 @@ lib.makeScope newScope (self: with self; {
   }) {};
 
   xf86inputlibinput = callPackage ({ stdenv, pkg-config, fetchurl, xorgproto, libinput, xorgserver }: stdenv.mkDerivation {
-    name = "xf86-input-libinput-1.0.0";
+    name = "xf86-input-libinput-1.0.1";
```
Comment (mweinelt) [inline pkgs/servers/x11/xorg/default.nix:1720]: Any particular reason we don't move to pname/version here?

**Review by mweinelt — APPROVED** [review]

File: pkgs/servers/x11/xorg/default.nix (hunk 1 above)
Comment (TredwellGit) [reply pkgs/servers/x11/xorg/default.nix:1720]: Someone would need to update https://github.com/NixOS/nixpkgs/blob/master/pkgs/servers/x11/xorg/generate-expr-from-tarballs.pl.

**Comment by lukegb** [comment]
@ofborg test xterm xfce plasma5

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 119831` at 286c1918 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>libsForQt512.plasma-desktop</li>
    <li>libsForQt514.plasma-desktop</li>
  </ul>
</details>
<details>
  <summary>4 packages built successfully:</summary>
  <ul>
    <li>libsForQt5.plasma-desktop (libsForQt515.plasma-desktop ,plasma5Packages.plasma-desktop)</li>
    <li>lxqt.lxqt-config</li>
    <li>xfce.xfce4settings (xfce.xfce4-settings)</li>
    <li>xorg.xf86inputlibinput</li>
  </ul>
</details>

**Review by lukegb — APPROVED** [review]

**Pull request merged**